tests/utils_tests/test_archive.py

```diff
@@ -42,6 +44,21 @@ def test_extract_function(self):
         extract(self.archive_path, self.tmpdir)
         self.check_files(self.tmpdir)
 
+    @unittest.skipIf(
+        sys.platform == 'win32',
+        'Python on Windows has a limited os.chmod() support.',
+    )
+    def test_extract_file_permissions(self):
+        """
+        Make sure permissions are kept after extraction from an archive.
```

Comment:
State the expected behavior without preambles like "Make sure" (as mentioned in [Python style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style)).